@navikt/core/css/darkside/accordion.darkside.css

```diff
@@ -115,67 +111,62 @@
 }
 
 /* ---------------------------- Accordion content --------------------------- */
-.navds-accordion__content {
-  --__acx-accordion-content-line-width: 2px;
-
-  padding-inline: var(--ax-space-24) var(--ax-space-12);
-  display: none;
-  overflow: hidden;
-  padding-block: 0;
-  max-height: 0;
-  border-color: transparent;
-  transition: all 250ms cubic-bezier(0.2, 0, 0, 1) allow-discrete;
-  animation: navds-accordion-content-animation 250ms cubic-bezier(0.2, 0, 0, 1);
-}
-
-.navds-accordion__item--no-animation {
-  animation: none;
-}
-
-@keyframes navds-accordion-content-animation {
-  0% {
-    opacity: 0.01;
-  }
-
-  20% {
-    opacity: 0.01;
-  }
-
-  100% {
-    opacity: 1;
-  }
-}
-
-.navds-accordion--indent > .navds-accordion__item > .navds-accordion__content {
-  padding-inline: var(--ax-space-24) var(--ax-space-12);
-}
 
 .navds-accordion--indent > .navds-accordion__item .navds-accordion__content-inner {
-  box-shadow: -2px 0 0 0 var(--ax-border-subtleA);
   padding-block: var(--ax-space-8);
-
-  /* Aligns content with heading-section */
   padding-inline: calc(var(--ax-space-20) + 2px) var(--ax-space-20);
 
   @media (forced-colors: active) {
     border-left: 1px solid canvastext;
   }
 }
 
-.navds-accordion__item[data-expanded="true"] > .navds-accordion__content {
-  display: block;
-  opacity: 1;
-  overflow: visible;
-  max-height: fit-content;
-  padding-block: var(--ax-space-8);
-  padding-block-end: var(--ax-space-24);
-  border-color: var(--ax-border-subtleA);
+.navds-accordion__item {
+  & > .navds-accordion__content {
+    display: grid;
+    box-shadow: -2px 0 0 0 var(--ax-border-subtleA);
```

Comment:
<img width="548" alt="Screenshot 2025-01-27 at 09 59 52" src="https://github.com/user-attachments/assets/b7698d06-f3d6-4e9d-8571-87b244704bfd" />

Left border should only be set when indent is defined above
```
Story-path
/?path=/story/ds-react-accordion--indent&globals=mode:darkside
```